config/core/test-configs.yaml

```diff
@@ -2025,6 +2025,13 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
```

Comment:
Tested OK: https://staging.kernelci.org/test/plan/id/623377cbedc070ee604b314a/
Many skip results are showing up but that's an artifact of how the skiplist works.